README.md

```diff
@@ -130,6 +130,34 @@ class RealAuthenticator @Inject constructor() : Authenticator
 abstract fun bindRealAuthenticator(authenticator: RealAuthenticator): Authenticator
 ```
 
+## Contributed multibindings
+
+Similar to contributed bindings `@ContributesMultibinding` will generate a multibindings method 
```

Comment:
Similar to contributed bindings, `@ContributesMultibinding will generate a multibindings method for (all/an) annotated class(s). Qualifiers are supported the same way as normal bindings.